tests/functional/metadata/mongodb/delObject.spec.js

```diff
@@ -0,0 +1,365 @@
+                mongodb: {
+                    replicaSetHosts: 'localhost:27018',
+                    writeConcern: 'majority',
+                    replicaSet: 'rs0',
+                    readPreference: 'primary',
+                    database: DB_NAME,
+                },
+            };
+            metadata = new MetadataWrapper(IMPL_NAME, opts, null, logger);
+            metadata.setup(done);
+        });
+    });
+
+    afterAll(done => {
+        async.series([
+            next => metadata.close(next),
+            next => mongoserver.stop()
+                .then(() => next())
+                .catch(next),
+        ], done);
+    });
+
+    beforeEach(done => {
+        const bucketMD = BucketInfo.fromObj({
+            _name: BUCKET_NAME,
+            _owner: 'testowner',
+            _ownerDisplayName: 'testdisplayname',
+            _creationDate: new Date().toJSON(),
+            _acl: {
+                Canned: 'private',
+                FULL_CONTROL: [],
+                WRITE: [],
+                WRITE_ACP: [],
+                READ: [],
+                READ_ACP: [],
+            },
+            _mdBucketModelVersion: 10,
+            _transient: false,
+            _deleted: false,
+            _serverSideEncryption: null,
+            _versioningConfiguration: null,
+            _locationConstraint: 'us-east-1',
+            _readLocationConstraint: null,
+            _cors: null,
+            _replicationConfiguration: null,
+            _lifecycleConfiguration: null,
+            _uid: '',
+            _isNFS: null,
+            ingestion: null,
+        });
+        metadata.createBucket(BUCKET_NAME, bucketMD, logger, err => {
+            if (err) {
+                return done(err);
+            }
+            collection = metadata.client.getCollection(BUCKET_NAME);
+            return done();
+        });
+    });
+
+    afterEach(done => {
+        metadata.deleteBucket(BUCKET_NAME, logger, done);
+    });
+
+    it('Should delete non versioned object', done => {
+        const bucketName = BUCKET_NAME;
+        const params = {
+            objName: 'non-deleted-object',
+            objVal: {
+                key: 'non-deleted-object',
+                versionId: 'null',
+            },
+            vFormat: 'v0',
+        };
+        const versionParams = {
+            versioning: false,
+            versionId: null,
+            repairMaster: null,
+        };
+        return async.waterfall([
+            next => {
+                // we put the master version of object
+                metadata.putObjectMD(BUCKET_NAME, params.objName, params.objVal,
+                versionParams, logger, err => {
+                    if (err) {
+                        return next(err);
+                    }
+                    return next();
+                });
+            },
+            next => {
+                // we put the master version of a second object
+                params.objName = 'object-to-deleted';
```

Comment:
typo, should be `object-to-delete`

```suggestion
                params.objName = 'object-to-delete';
```

Reply:
fixed in https://github.com/scality/Arsenal/pull/1729/commits/a6f48b9f342344e6df5ca2e3b7a2d03d8737fbba